docs/Unicode.md

```diff
@@ -1,47 +1,81 @@
+Besides the limited set of latin fonts built into the PDF format, Fpdf2 offers full support for using and embedding Unicode (TrueType "ttf" and OpenType "otf") fonts. To keep the output file size small, it only embeds the subset of each font that is actually used in the document. This part of the code has been completely rewritten since the fork from PyFPDF. It uses the [fonttools](https://fonttools.readthedocs.io/en/latest/) library for parsing the font data, and [harfbuzz](https://harfbuzz.github.io/) (via [uharfbuzz](https://github.com/harfbuzz/uharfbuzz)) for [text shaping](TextShaping.html).
 
-Before you can use UTF-8, you have to install at least one Unicode font in the 
-font directory (or system font folder). Some free font packages are available 
-for download (extract them into the font folder):
+To make use of that functionality, you have to install at least one Unicode font, either in the system font folder or in some other location accessible to your program.
+For professional work, many designers prefer commercial fonts, suitable to their specific needs. There are also many sources of free TTF fonts that can be downloaded online and used free of cost (some of them may have restrictions on commercial redistribution, such as server installations or including them in a software project).
 
-  * [DejaVu](http://dejavu-fonts.org/) family: Sans, Sans Condensed, Serif,
-Serif Condensed, Sans Mono (Supports more than 200 languages)
+  * [Font Library](https://fontlibrary.org/) - A collection of fonts for many languates with an open source type license.
+
+  * [Google Fonts](https://fonts.google.com/) - A collection of free to use fonts for many languages.
+
+  * [Microsoft Font Library](https://learn.microsoft.com/en-gb/typography/font-list/) - A large collection of fonts that are free to use.
+
+  * [GitHub: Fonts](https://github.com/topics/fonts) - Links to public repositories of open source font projects as well as font related software projects.
 
   * [GNU FreeFont](http://www.gnu.org/software/freefont/) family: FreeSans,
 FreeSerif, FreeMono
 
-  * [Indic](http://en.wikipedia.org/wiki/Help:Multilingual_support_(Indic))
-(ttf-indic-fonts Debian and Ubuntu package) for Bengali, Devanagari, Gujarati,
-Gurmukhi (including the variants for Punjabi), Kannada, Malayalam, Oriya,
-Tamil, Telugu, Tibetan
+To use a Unicode font in your program, use the [`add_font()`](fpdf/fpdf.html#fpdf.fpdf.FPDF.add_font), and then the  [`set_font()`](fpdf/fpdf.html#fpdf.fpdf.FPDF.set_font) method calls.
+
+
+### Built-in Fonts vs. Unicode Fonts ###
+
+The PDF file format knows a small number of "standard" fonts, namely "courier", "helvetica", "times", "symbol", and "zapfdingbats". The first three are available in regular, bold, italic, and bold-italic versions. Any PDF processor (eg. a viewer) must provide those fonts for display. To use them, you don't need to call `.add_font()`, but only `.set_font()`.
+
```

Comment:
Do you think it's work adding:

On fpdf, "arial" is an alias for the standard font "helvetica".

Reply:
Hmmm.... Didn't we want to remove those aliases?

Reply:
We can probably deprecate, removing it would break programs for many of our users.

Reply:
Given that it was never documented, I'm not really sure, but that's a topic for a different PR.